src/Razor/src/Microsoft.AspNetCore.Razor.Language/ComponentResources.resx

```diff
@@ -123,14 +123,20 @@
   <data name="BindTagHelper_Element_Documentation" xml:space="preserve">
     <value>Binds the provided expression to the '{0}' attribute and a change event delegate to the '{1}' attribute.</value>
   </data>
+  <data name="BindTagHelper_Element_Event_Documentation" xml:space="preserve">
```

Comment:
Need halpp with documentation and error messages.

Reply:
> Specifies the event handler name to attach for change notifications.